### PR TITLE
Bloat Control

### PR DESCRIPTION
**Pull request by trieloff**
Cloudflare truncates all log messages that exceed 16kb. This should normally not be a problem,
but in some cases we have seen truncation messages. This PR will apply a couple of tweaks to
the logged JSON object to keep size in check:
1. limit all string properties to 1024 characters
2. cast most CWVs to integers, as we don't need the fake accuracy nor the floating point length mismatches

- **feat(utils): add bloat control utility to ensure JSON strings don't get too long**
- **feat(logger): control bloat of JSON logs for Cloudflare**

### CODE REVIEW

File: src/utils.mjs [hunk 1]
```diff
@@ -276,3 +276,38 @@ export function getSubsystem(req) {
...
+ * @returns {String} the sanitized object as a JSON string
+ */
+export function bloatControl(obj) {
+  // the object can have nested objects, so we may need recursion
+  const sanitize = (o, k) => {
+    if (typeof o === 'string' && o.length > 1024) {
+      return `${o.substring(0, 1024)}…`;
+    }
+    if (typeof o === 'number') {
+      if (['LCP', 'INP', 'TTFB'].includes(k)) {
```
Comment (maxakuru) [inline src/utils.mjs:296]: `time` could be included in this, since `performance.now` in v2 will give us floats but we probably don't care?

Comment (maxakuru) [reply src/utils.mjs:296]: also maybe worth fixing the decimal places of `CLS`, maybe to 4 or 5? It seems like pagespeed shows at most 3

**Review by maxakuru — APPROVED** [review]
🚀

**Comment by bosschaert** [comment]
LGTM!

**Pull request merged**